alerts/machine-config-operator/ExtremelyHighIndividualControlPlaneMemory.md

```diff
@@ -0,0 +1,27 @@
+# ExtremelyHighIndividualControlPlaneMemory
+
+## Meaning
+
+This alert fires when the memory usage per an instance within control plane
+nodes exceeds 90% of total memory available per a node.  
+This is a critical level alert.
+
+## Impact
+
+At such very low free memory, the cluster becomes unstable and expected slow
+responses from kube-apiserver or failing requests specially on etcd.
+Moreover, OOM kill is expected which negatively influences the pod scheduling.
+
+## Diagnosis
+
+To fix this, increase memory of the affected node of control plane nodes.
+
+## Mitigation
+
+Using `HighOverallControlPlaneMemory` alert you can mitigate this situation.
```

Comment:
s/Using/Responding to a 
s/you can/can (if you accept the above comment)  (Responding to a `HighOverallControlPlaneMemory` alert can mitigate...)

Reply:
+1